app/upgrades/v4_1/constants.go

```diff
@@ -0,0 +1,22 @@
+package v4_1
+
+import (
+	store "github.com/cosmos/cosmos-sdk/store/types"
+	"github.com/terpnetwork/terp-core/v4/app/upgrades"
+	clocktypes "github.com/terpnetwork/terp-core/v4/x/clock/types"
+	driptypes "github.com/terpnetwork/terp-core/v4/x/drip/types"
+)
+
+// UpgradeName defines the on-chain upgrade name for the Terp v4 upgrade.
+const UpgradeName = "v4.1"
+
+var Upgrade = upgrades.Upgrade{
+	UpgradeName:          UpgradeName,
+	CreateUpgradeHandler: CreateV4_1UpgradeHandler,
+	StoreUpgrades: store.StoreUpgrades{
+		Added: []string{
+			driptypes.ModuleName,
```

Comment:
did v4 get into prod with the burn module? if so make sure to Remove that store key here, not just remove the keeper

Reply:
Appreciate you following up here! The burn module only made it to our test network `90u-2`. Is it certain the burn module needs the store key removed? We do not have a store key specified [here](https://github.com/terpnetwork/terp-core/blob/v4.0.0/x/burn/burner.go), and we also did not specify the module stored in the [v4 upgrade](https://github.com/terpnetwork/terp-core/blob/v4.0.0).

Reply:
ahh good call. was a busy day and forgot it's more of a plugin vs module

so my comment does no apply, you're all good

Reply:
no worries, appreciate you!